### PR TITLE
Prettier formatting

### PR DESCRIPTION
**Pull request by alvaromb**
Added prettier formatting and a prettier script to package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,15 +25,14 @@
     "tcomb": "^3.0.0"
   },
   "devDependencies": {
+    "@types/eslint-plugin-prettier": "^2.2.0",
+    "@types/prettier": "^1.7.0",
```
Comment (gcanti) [inline package.json]: Just curious, what's this for? `@types/eslint-plugin-prettier`, `@types/prettier`

Comment (alvaromb) [reply package.json]: These are Flow types auto-installed by an extension of VSCode. I'll remove them.

**Comment by alvaromb** [comment]
There's an issue I still need to solve with ESLint, I'll ping you when done.

**Comment by alvaromb** [comment]
@gcanti changed a few things about the Travis config, but now the code is formatted by `prettier` and tests successfully pass!

**Pull request merged**